src/components/calculator/PointsDisplay.tsx

```diff
@@ -29,49 +35,296 @@ function displayPercent(points: number): string {
+              their risk:
+            </h4>
+            <ol className="mt-2">
+              <li>
+                Talk to them about how their choices affect your risk, as well
+                as theirs
+              </li>
+              <li>
+                Ask them to <abbr title="Masked, Ourdoors, Distanced">MOD</abbr>
+                ify their activities (Masked, Outdoors, Distanced)
+              </li>
+              <li>
+                Essential workers can wear a top quality mask (e.g., N95 or
+                KN95) to work
+              </li>
+              <li>
+                Ask them to limit undistanced socializing to as few people as
+                possible
+              </li>
+              <li>
+                Encourage close contacts to isolate and get tested at the first
+                sign of COVID symptoms
+              </li>
+              <li>
+                If they have high COVID risk and you must be inside with them:
+                <ul>
+                  <li>Wear masks inside</li>
+                  <li>
+                    Maximize ventilation using open windows, fans blowing in
+                    outside air, and HEPA filters
+                  </li>
+                  <li>
+                    Isolate within the house by staying in different rooms.
+                  </li>
+                </ul>
+              </li>
+            </ol>
+          </>
+        )}
+      </div>
+    </>
+  )
 
   return (
     <Card>
-      <h2>How risky is this?</h2>
-      <GenericSelectControl
-        id="budget-selector"
-        label="If your risk tolerance is..."
-        popover={riskTolerancePopover}
-        setter={(e: string) => props.riskBudgetSetter(Number.parseInt(e))}
-        value={props.riskBudget}
-        source={{
-          '10000': {
-            label: '1% per year (suggested if not at increased risk)',
-            multiplier: 1,
-          },
-          '1000': {
-            label:
-              '0.1% per year (suggest if at increased risk or regularly interacting with people at increased risk)',
-            multiplier: 0.1,
-          },
-        }}
-      />
-      <p className="readout">
-        ... then for you this is a{' '}
-        <span className={activeRiskLevel.style}>
-          <b>{showPoints(points) ? activeRiskLevel.title : '——'}</b>
-        </span>{' '}
-        risk activity.
-      </p>
-      <h2>What does this mean numerically?</h2>
-      <p>
-        This is a roughly ~{displayPoints(points)}-in-a-million (
-        {displayPercent(points)}){props.repeatedEvent ? ' per week ' : ' '}
-        chance of getting COVID from this activity with these people.
-      </p>
-      <p>{budgetConsumption(points, props.riskBudget)}</p>
+      {showPoints(props.points) ? calculationBreakdown : ''}
+
+      <Form.Group>
+        <GenericSelectControl
+          id="budget-selector"
+          label=" "
```

Comment:
Can you just leave off the label? Or do you need the space?

Reply:
fixed with an update on GenericSelectControl